docs/tutorials/istio.md

```diff
@@ -1,12 +1,12 @@
 # Configuring ExternalDNS to use the Istio Gateway Source
-This tutorial describes how to configure ExternalDNS to use the Istio Gateway source.
+This tutorial describes how to configure External-DNS to use the Istio Gateway source.
```

Comment:
I think that's actually not a typo, we spell it `ExternalDNS` even in the readme: https://github.com/kubernetes-sigs/external-dns#externaldns